### ISSUE TITLE
Sub articles

### ISSUE THREAD

**Issue by josephgupkim** [Coralie9/f1-3-c2p1-colmar-academy#2]
The sub-articles on the right hand side seemed to drop too low (on my screen anyway). It seemed that taking this property off corrected it. But remember it should be stretching the full length from top to bottom.

.flex-container {
   align-items: center;
}

**Comment by Coralie9** [comment]
Yes they were dropping too low. I tried to correct it but I did not manage. Thank you for your help!
